### DISCUSSION TITLE
Need to dial back activities at the project

### DISCUSSION THREAD

**Discussion by Grounded0** [dosbox-staging/dosbox-staging#3807] [General]
My schizoaffective disorder has been getting much worse again and I need to concentrate on getting well. I'm planning on moving to Spain to better manage my mental health due to mild seasonal changes in the weather and the amount of light than at 60 degrees north here in Finland.

That's why I need to cut back on project funding for now because my disability pension isn't enough for funding living in Spain and funding this project at the same time. I will concentrate on getting us on Debian official package repository but that's about the maximum I have bandwidth for at the moment. 🙏

**Comment by johnnovak** [comment]
@Grounded0 I hope you'll feel better in Spain, man. It's a nice sunny country, and it's much more pleasant to live there than in Northern Finland, that's for sure. I always felt slightly depressed during the winter months back in Hungary, and that's nowhere near Finland levels of darkness, so I can certainly sympathise 😆 

Let me know if you want me to revoke your admin privileges if that makes you feel more relaxed (I remember you asked us to revoke your admin rights in the past, that's why I'm bringing it up).

I wish you all the best, and play some nice DOS games to relax! 😏 ❤️
